requirements/requirements-python3.6.txt

```diff
@@ -67,9 +68,9 @@ beautifulsoup4==4.7.1
 billiard==3.6.3.0
 black==19.10b0
 blinker==1.4
-boto3==1.12.39
+boto3==1.12.41
 boto==2.49.0
-botocore==1.15.39
+botocore==1.15.41
```

Comment:
Should those changes be part of this PR?

Reply:
I might want to split the change to several pieces - haven't decided yet, I was waiting for comments. But in general yes - this change should be part of this change. Whenever we change setup.py we will run generate-requiremens command which will (locally) run eager update strategy and potentially bump-up some unrelated requirements. Note that it does not change setup.py, it does not change dependencies it simply pins the latest version of the libraries in requirements.txt and it will run all tests against that version. 

This is actually very good because it reflects what "first time users will experience" - i.e. when you instal airflow for the first time this is what you will get - the latest requirements matching constraints specified in setup.py. This way anyone changing setup.py will automatically without even thinking about it testing if latest released dependent libraries are working fine. And if they will, it will be merged to master and will become the "good" version of pinned requirements

This way we do not have to remember to periodically tests if latest requirements still work and we do not need a separate "upgrade to latest requirements" commit. Remember - those files are not real requirements (we have them in setup.py) - those are snapshots of dependency versions that are confirmed to be working with Airflow.

So short answer - yes. I think it should be part of the change :)